src/harness/testcases/grant_testcase.py

```diff
@@ -114,33 +115,41 @@ def test_10_7_4_1_1_1_1(self):
     self.assertTrue(response['grantId'])
     self.assertEqual(response['channelType'], 'GAA')
     self.assertEqual(response['response']['responseCode'], 0)
+    del request, response
 
-  @winnforum_testcase
-  def test_10_7_4_1_3_1_2_1(self):
-    """CBSD grant request when CBSD ID does not exist in SAS.
-    CBSD sends grant request when its CBSD Id is not in SAS. The response
-    should be FAIL.
-    """
+    # Category B CBSD
```

Comment:
I wonder if it makes more sense to have a 'for' loop? It looks like the steps are exactly the same and that way we would make sure they don't fall out of sync.